extract/Dockerfile

```
@@ -30,12 +30,12 @@ RUN cd /extract-build/sitn_geoshop_connector/ && git checkout "${GEOSHOP_CONNECT
     cp /extract-build/extract-task-exec/target/original-extract-task-exec-*.jar /extract-build/extract/extract/src/main/resources/task_processors/extract-task-exec.jar
 
 RUN  cd /extract-build/extract/ && git checkout "${EXTRACT_VERSION}" && \
```

```suggestion
RUN cd /extract-build/extract/ && git checkout "${EXTRACT_VERSION}" && \
```

Just so it looks a bit prettier and since you also changed the lines below ;-)